tools/harvest_source_import/remote_ckan/lib.py

```diff
@@ -75,7 +81,7 @@ def list_harvest_sources(self, source_type=None, start=0, page_size=100, limit=0
             logger.info(f'  [{source_type}] Harvest source: {title} [{state}]')
             if state == 'active':
                 # We don't get full harvest soure info here. We need a custom call
-                yield self.get_full_harvest_source(hs)
+                yield self.get_full_harvest_source(hs)                
```

Comment:
some trailing whitespace here that you probably don't need :-)

Reply:
Yes, you are right. Fixed [here](https://github.com/GSA/catalog.data.gov/pull/78/commits/42ef6d80f0604512ae273a1419e32c03b8742d66)

Reply:
fixed